app/src/common/shared/org/mozilla/vrbrowser/browser/engine/Session.java

```diff
@@ -172,6 +170,13 @@ protected void shutdown() {
             }
         }
 
+        if (mSessionRemoved) {
+            mSessionChangeListeners.forEach(listener -> {
+                listener.onSessionRemoved(mState.mId);
+            });
+            mSessionRemoved = true;
```

Comment:
why set mSessionRemoved if it's already true?

Reply:
I think mSessionRemoved is never set to true?

Reply:
I am also looking for where `mSessionRemoved ` be set to true

Reply:
good catch, let me fix it.